src/mjolnir/pbfgraphparser.cc

```diff
@@ -2387,10 +2387,12 @@ struct graph_callback : public OSMPBF::Callback {
         if (tokens.size() == 2 && tmp.size()) {
 
           uint16_t mode = 0;
-          if (boost::algorithm::starts_with(tag_.first, "motorcar:conditional") ||
-              boost::algorithm::starts_with(tag_.first, "motor_vehicle:conditional")) {
+          if (boost::algorithm::starts_with(tag_.first, "motor_vehicle:conditional")) {
             mode = (kAutoAccess | kTruckAccess | kEmergencyAccess | kTaxiAccess | kBusAccess |
                     kHOVAccess | kMopedAccess | kMotorcycleAccess);
+          } else if (boost::algorithm::starts_with(tag_.first, "motorcar:conditional")) {
+            mode = (kAutoAccess | kTruckAccess | kEmergencyAccess | kTaxiAccess | kBusAccess |
+                    kHOVAccess);
```

Comment:
Oh sorry, for some reason my eyes didn't see that you're doing that here.. Anyways, I think this is too broad. This is only valid for `kTimeDenied`. for `kTimeAllowed` we don't need to add anything I think:

```suggestion
          } else if (boost::algorithm::starts_with(tag_.first, "motorcar:conditional")) {
          if (type == kTimedDenie) {
             mode = (kAutoAccess | kTruckAccess | kEmergencyAccess | kTaxiAccess | kBusAccess |
                      kHOVAccess);
           }
```

Reply:
I will follow you suggestion. (I only know how to drive motorcycle LoL)